_submissions/round-12/0/2015-02-18-about-april-clyburne-sherin.md

```diff
@@ -0,0 +1,11 @@
+---
+date: 2015-02-18
+round: Round 12
+title: About April Clyburne-Sherin
+author: April Clyburne-Sherin
+permalink: /2015/02/april-clyburne-sherin/
+tags:
+  - Biography
+---April Clyburne-Sherin is an epidemiologist at The Hospital for Sick Children in Toronto. She works in the [EnRICH Lab](http://www.sickkids.ca/Research/EnRICH/About-Us/index.html) where her research aims to improve the quality and utility of clinical trials. She is an Open Science advocate and a lover and maker of [radio](https://pioneerradio.wordpress.com/). April attended a Software Carpentry course to learn how to use version control for her R code. The course was so valuable that she joined the Software Carpentry Instructor Course to help other researchers learn the skills to create reproducible, shareable, and open science. She loves her bike, strong coffee, and ice skating at night.
```

Comment:
Hi April,
Thanks for the bio, but in order to format properly, the three dashes that close the header have to be on a line of their own - please insert a line break between the '---' and the start of your name.  You can do this to the file in your repository on your desktop, then push again to your repository on GitHub, and this pull request will automatically update.
Thanks,
Greg